Tests/SwiftParserTest/translated/ForwardSlashRegexTests.swift

```diff
@@ -1689,14 +1729,22 @@ final class ForwardSlashRegexTests: ParserTestCase {
       _ = ℹ️/1️⃣               2️⃣
       """,
       diagnostics: [
-        DiagnosticSpec(locationMarker: "1️⃣", message: "bare slash regex literal may not start with space"),
+        DiagnosticSpec(
+          locationMarker: "1️⃣",
+          message: "bare slash regex literal may not start with space",
+          fixIts: [
+            "convert to extended regex literal",
+            #"insert '\'"#,
+          ]
+        ),
         DiagnosticSpec(
           locationMarker: "2️⃣",
           message: "expected '/' to end regex literal",
           notes: [NoteSpec(message: "to match this opening '/'")],
           fixIts: ["insert '/'"]
         ),
       ],
+      applyFixIts: ["insert '/'"],
```

Comment:
Typo? The Fix-It is `insert '\'` but here you’re using a forward slash instead of a backslash. Same in the other tests as well.

Reply:
I decided to silence the new fix-its for all translated test cases because I thought these test cases weren't meant for testing these new fix-its?

Reply:
As a side note, I've duplicated the `assertParse` statement for each test case in `RegexLiteralTests` affected by this PR so as to apply each of the two fix-its at a time, e.g.
```swift
assertParse(
  """
  let x = /1️⃣ a/
  """,
  diagnostics: [
    DiagnosticSpec(
      message: "bare slash regex literal may not start with space",
      fixIts: [
        "convert to extended regex literal with '#'",
        #"insert '\'"#,
      ]
    )
  ],
  applyFixIts: ["convert to extended regex literal with '#'"],
  fixedSource: """
    let x = #/ a/#
    """
)

assertParse(
  """
  let x = /1️⃣ a/
  """,
  diagnostics: [
    DiagnosticSpec(
      message: "bare slash regex literal may not start with space",
      fixIts: [
        "convert to extended regex literal with '#'",
        #"insert '\'"#,
      ]
    )
  ],
  applyFixIts: [#"insert '\'"#],
  fixedSource: #"""
    let x = /\ a/
    """#
)
```

It's become a bit unwieldy.

Reply:
Ah, sorry. I think I realized the same later on as well but then forgot to remove my comment again. Your change makes perfect sense